uat-vocabularies/geological-observation-type.ttl

```diff
@@ -611,6 +611,15 @@ obstp:younging a skos:Concept ;
     skos:prefLabel "Younging"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/geological-observation-type> .
 
+obstp:lot-adjacent a skos:Concept ;
+    skos:definition "Is the lot-on-plan number used to describe a feature not within the bounds of that parcel, but instead, adjacent to."@en ;
+    rdfs:isDefinedBy <http://linked.data.gov.au/def/geological-observation-type> ;
+    skos:inScheme <http://linked.data.gov.au/def/geological-observation-type> ;
+    skos:notation "LA" ;
+    skos:prefLabel "Lot Adjacent"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/geological-observation-type> .
```

Comment:
Need some clarity on the purpose of this.

Reply:
@GSQ-AI  are you able to help? I'm not up to date with quarries